### PR TITLE
Yarn instead of npm and add readme

### PR DESCRIPTION
**Pull request by chavlji**
Emphasize the use of **yarn** everywhere instead of **npm**.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -10,13 +10,13 @@ jobs:
       - checkout
       - restore_cache:
           keys:
-          - v1-npm-{{ checksum "yarn.lock" }}
-          - v1-npm-
-      - run: npm install
+          - v1-yarn-{{ checksum "yarn.lock" }}
+          - v1-yarn-
```
Comment (mplesec) [inline .circleci/config.yml:14]: Probably a typo?

File: .circleci/config.yml [hunk 2]
```diff
@@ -38,14 +38,14 @@ jobs:
       - checkout
       - attach_workspace:
           at: .
-      - run: npm run test
+      - run: yarn test
   flow:
```
Comment (mplesec) [inline .circleci/config.yml:42]: Aren't we using typescript?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,33 @@
-# zen-ui
-A collection of UI components that are shared across Reciprocity applications
+# ZEN-UI
+*A collection of UI components that are shared across Reciprocity applications*
+
+## Dev environment setup
+`yarn`
+`yarn start` This command wil concurrently run both Stencil and Storybook in watch mode.
```
Comment (mplesec) [inline README.md:6]: typo: wil

File: README.md [hunk 4]
```diff
@@ -1,2 +1,33 @@
...
+Library is based on [Stencil](https://stenciljs.com/docs/introduction). Stencil is a lib to produce standard **web components**.
+
+### Web components
+Web components are part of the existing Web standards. Thus they're natively supported by modern web browsers (IE with polyfills). Therefore they can be consumed in any FE framework (React, Vue, Angular,...) or even in Vanilla js.
+
+### Storybook
+For presentation we're using [Storybook](https://storybook.js.org/). Storybook is an industry standard for presenting component libraries and design styles.
+Stencil support is under development since March 2019 (1.5 year since this writing), but there's nothing public yet.
+Therefore this lib is based on Storybook config for web components. Setup was created from ground up, but was modeled on [dutscher/stencil-storybook](https://github.com/dutscher/stencil-storybook).
+Under the hood it uses **lit-html** for rendering the components. This is why components in each story is implemented using lit-html. Lit-html allows us to use some powerfull stuff that are common for higher FE frameworks, like binding non-scalar props (arrays, objects) directly and listening to emitted events.
```
Comment (mplesec) [inline README.md:28]: typo: powerfull

File: README.md [hunk 5]
```diff
@@ -1,2 +1,33 @@
...
+
+## Dev environment
+
+### Components documentation
+Documentation for each component is auto-generated by stencil. Check [Stencil Docs Readme generation](https://stenciljs.com/docs/docs-readme) for detailed guides on how to tailor docs using special code comments. We pick this autogenerated `readme.md` of each component and use it inside the Storybook.
```
Comment (mplesec) [inline README.md:33]: So is it `auto-generated` or `autogenerated`? 😅

**Review by mplesec — APPROVED** [review]
A few minor things, feel free to merge and address in a follow-up PR.

File: .circleci/config.yml (hunk 2 above)
Comment (chavlji) [reply .circleci/config.yml:41]: We are. But what does it have to do with the test command?

**Review by mplesec — APPROVED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (mplesec) [reply .circleci/config.yml:41]: Comment was about the `flow` job.

**Review by f-jauregui — APPROVED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (chavlji) [reply .circleci/config.yml]: Oh, you're right! We don't even have a `flow` script defined. I'll remove it.

**Pull request merged**